### PR TITLE
Add `Building an Electron Edge` book

### PR DESCRIPTION
**Pull request by MaxGfeller**
I'm one of the authors of Building an Electron Edge and i feel like the book is a good introduction for starters.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Do you have a place people can read a preview of a chapter or something? I don't think anyone will just blindly buy a book without being able to judge the quality.

**Comment by MaxGfeller** [comment]
Sure! It's available on [Oreilly](http://shop.oreilly.com/product/9781939902344.do) and on Amazon as [an ebook](https://www.amazon.com/Developing-Electron-Edge-edge-Book-ebook/dp/B01G7TTKSK#nav-subnav) and [print version](https://www.amazon.com/Developing-Electron-Edge-Adam-Lynch/dp/1939902398/). All of them offer a small preview.

File: readme.md [hunk 1]
```diff
@@ -280,6 +280,10 @@ Made with Electron.
 - [Electron Rocks!](http://electron.rocks) - Blog about working with Electron.
 
 
+## Books
+
+- [Building an Electron Edge](https://bleedingedgepress.com/developing-an-electron-edge/)
```
Comment (sindresorhus) [inline readme.md]: Add a preview link at the the end:

```
[(Preview)](https://read.amazon.com/kp/embed?asin=B01G7TTKSK&asin=B01G7TTKSK&preview=newtab&linkCode=kpe&ref_=cm_sw_r_kb_dp_DLhOxb0XZ3MEC)
```

File: readme.md [hunk 2]
```diff
@@ -280,6 +280,10 @@ Made with Electron.
 - [Electron Rocks!](http://electron.rocks) - Blog about working with Electron.
 
 
+## Books
```
Comment (sindresorhus) [inline readme.md]: Needs to be added to the Table of Contents

**Comment by MaxGfeller** [comment]
Alright, updated the PR. I'll also ask our publisher to update their site for the book with links to the print version.

**Comment by sindresorhus** [comment]
> Alright, updated the PR. I'll also ask our publisher to update their site for the book with links to the print version.

I would also ask them to add a sample, like on Amazon.

**Comment by MaxGfeller** [comment]
You're absolutely right. Thanks for accepting the PR!

**Pull request merged**